signalwire/signalwire-legacy-js
# Publishing script

The code in this changeset includes a new CLI tool for publishing packages to npm.

### How to test it:
Publishing is always running with `--dry-run` so it's totally safe to run the following set of steps.

1. Make sure there's at least one pending changeset inside of the `.changeset` folder. The script won't work unless there's at least one.
2. From the root, run `npm run release:dev`

### TODO:

- [x] Add ability to run the entire test suite before publishing
- [x] Add ability to publish `dev` releases
  - [x] Create unique dev hashes per release
  - [x] Publish the package
- [ ] Add ability to publish `production` releases
